authors/apps/article/urls.py

```diff
@@ -13,5 +13,8 @@
     path('articles/<slug>/dislike', DislikeAPIView.as_view()),
     path('articles/<slug>/comments/', CommentsAPIView.as_view()),
     path('articles/<slug>/comments/<pk>', RetrieveCommentsAPIView.as_view()),
-    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view())
+    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view()),
+    path('articles/<slug>/like_unlike_comment/<pk>', LikeUnlikeAPIView.as_view()),
```

Comment:
Hey @mnswaleh we can leave this as `articles/<slug>/like_comment/<pk>'

Reply:
Ok